core/src/processing/core/PShapeSVG.java

```diff
@@ -406,16 +439,13 @@ protected PShape parseChild(XML elem) {
     } else if (name.equals("pattern")) {
       PGraphics.showWarning("Patterns are not supported.");
 
-    } else if (name.equals("stop")) {
```

Comment:
Don't remove this, there's documentation here for each item. If we wanted it to be a single if entry, we'd have done it that way in the first place.